auto_round/script/mllm.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
         self.add_argument("--asym", action='store_true',
                             help=" asym quantization")
 
-        self.add_argument("--dataset", type=str, default=None,
+        self.add_argument("--dataset", type=str, default="llava",
```

Comment:
better give more detail info, because we may add llama-instruct and llama-pretrained later

Reply:
llava_v1_5_mix665k?